pkgs/games/anki/default.nix

```diff
@@ -31,9 +32,9 @@ stdenv.mkDerivation rec {
     pythonPath = [ pyqt4 sqlalchemy pyaudio beautifulsoup httplib2 ]
               ++ lib.optional plotsSupport matplotlib;
 
-    buildInputs = [ python wrapPython lame mplayer libpulseaudio ];
+    buildInputs = [ python wrapPython lame mplayer libpulseaudio pytest glibcLocales nose ];
 
-    phases = [ "unpackPhase" "patchPhase" "installPhase" ];
+    phases = [ "unpackPhase" "patchPhase" "testPhase" "installPhase" ];
```

Comment:
if the `fixupPhase` does not do any harm here, please remove the explicit phases selection here.